### ISSUE TITLE
Overwrites my runtime config

### ISSUE THREAD

**Issue by mariusbolik** [YsarocK/nuxt-appwrite#60]
**Describe the bug**
Hello, thank you for this library. I noticed, that my runtime config gets overwritten. I can't use any other variables from my runtime config in my app.

**To Reproduce**
```
 const config = useRuntimeConfig();
 console.log('CONFIG', config)
```

**Expected behavior**
It should show my whole runtime config object, but it only shows an appwrite object. All other information is gone.

**Screenshots**
<img width="719" alt="Bildschirmfoto 2023-07-20 um 09 54 37" src="https://github.com/YsarocK/nuxt-appwrite/assets/39862171/12347359-f5b8-4d3a-8b20-c1718a4a3593">
<img width="582" alt="Bildschirmfoto 2023-07-20 um 09 51 50" src="https://github.com/YsarocK/nuxt-appwrite/assets/39862171/e1cb5dbf-3806-4db4-af86-14bcf4bfa093">

**Comment by YsarocK** [comment]
Hi,

Normally, just fixed that in the [v1.6.4](https://github.com/YsarocK/nuxt-appwrite/releases/tag/v1.6.4).

Thanks for reaching me,
Etienne

**Comment by mariusbolik** [comment]
Hi,

I am on 1.6.4!
